airflow/cli/commands/variable_command.py

```diff
@@ -76,7 +78,29 @@ def variables_import(args):
 
 def variables_export(args):
```

Comment:
Would be better if we treat the absence of value in `args.file`  as an indication to use stdout instead of equating the name with `args.file.name == "<stdout>"`. WDYT?